ulib/axlibc/include/sched.h

```diff
@@ -27,6 +27,9 @@ typedef struct cpu_set_t {
 #define CPU_SET(i, set) CPU_SET_S(i, sizeof(cpu_set_t), set);
 #define CPU_ZERO(set)   CPU_ZERO_S(sizeof(cpu_set_t), set)
 
+#define FSHIFT  11            /* nr of bits of precision */
+#define FIXED_1 (1 << FSHIFT) /* 1.0 as fixed-point */
```

Comment:
Are these two parameters from musl libc or glibc? All header files in axlibc/include should follow musl libc.